beacon-chain/sync/rpc_send_request.go

```diff
@@ -453,6 +453,10 @@ func SendDataColumnSidecarsByRangeRequest(
 	// Send the request.
 	stream, err := p.P2P.Send(p.Ctx, request, topic, pid)
 	if err != nil {
+		if p.DownscorePeerOnRPCFault {
```

Comment:
can peers only be downscored for 1 reason? wondering if a better design would be to collect all the faults, then loop downscore clearing the collection if it can happen more than once.